test/data/kb_uploadmethods.import_fastq_interleaved_as_reads_from_staging.spec.json

```diff
@@ -0,0 +1,287 @@
+{
+    "info": {
```

Comment:
this is a popular app for testing -- we also have the spec in `src/biokbase/narrative/tests/data/app_spec.json`!

Reply:
Yeah, I'd like to consolidate those at some point.